src/keyring/KeyringController.test.ts

```diff
@@ -247,6 +346,36 @@ describe('KeyringController', () => {
     expect(account).toBe(recovered);
   });
 
```

Comment:
For method below `signPersonalMessage` it seems not to fail for empty data, I am not sure if that is ok.

Reply:
Created issue https://github.com/MetaMask/controllers/issues/799

Reply:
🤔